pkg/chunk/cached_store.go

```diff
@@ -614,6 +622,34 @@ type Config struct {
 	Prefetch       int
 }
 
+func (c *Config) ValidateParams() {
+	if !c.Writeback && c.DelayUpload != "" {
+		logger.Fatalf("writeback should be set true before congiure deploy-upload")
```

Comment:
The DelayUpload will be ignore if writeback is false, so Warning() should be better than Fatalf()

Reply:
Done!